webpack/webpack.dev.babel.js

```diff
@@ -8,37 +8,51 @@ import webpackConfig from './webpack.config.babel';
 
 // Defaults to value in package.json.
 // Can override with npm config set dcos-ui:port 80
```

Comment:
This looks good to me. I'm going to migrate our `externalplugins` config back to using `dcos-ui:` prefix but have to find some time one evening to investigate the options in CI. Don't think it will be a problem. Lets stick to using `dcos-ui:` prefix for the port.